source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
```

Comment:
Fails `yarn lint`: `error    Unexpected space before the ':' @typescript-eslint/type-annotation-spacing`